src/components/DetailsMetrics/DetailsMetrics.js

```diff
@@ -122,13 +188,61 @@ const DetailsMetrics = ({ selectedItem }) => {
     )
   }, [dispatch, selectedItem])
 
+  useEffect(() => {
+    const selectedDate = detailsStore.dates.selectedOptionId
+    if (!selectedDate || !(selectedDate in timeRangeMapping)) return
+
+    setSelectedDate(timeRangeMapping[selectedDate])
+  }, [detailsStore.dates.selectedOptionId])
+
+  const fetchData = useCallback(
+    (params, params2, selectedItem) => {
+      metricsValuesAbortController.current = new AbortController()
+
+      return Promise.all([
+        dispatch(
+          detailsActions.fetchModelEndpointMetricsValues(
+            selectedItem.metadata.project,
+            selectedItem.metadata.uid,
+            params,
+            metricsValuesAbortController.current.signal
+          )
+        ),
+        dispatch(
+          detailsActions.fetchModelEndpointMetricsValues(
+            selectedItem.metadata.project,
+            selectedItem.metadata.uid,
+            params2,
+            metricsValuesAbortController.current.signal
+          )
+        )
+      ])
```

Comment:
it doesn't work properly because the loader is hidden when one of the request is   finished

Reply:
moreover it overrides values in the Redux

Reply:
move   dispatch(detailsActions.fetchEndpointMetricsValuesSuccess()) to prommis.all.